vmdb/app/views/layouts/_edit_to_email.html.haml

```diff
@@ -12,9 +12,11 @@
             = _("None")
           - else
             - if @temp[:email_to].present?
-              - emails = @temp[:email_to].sort { |a, b| a[1] <=> b[1] }.collect { |e| e[0] }
+              - emails = @temp[:email_to]
+                  .sort_by { |_email, display_email| display_name }
+                  .collect { |email, _display_email| email }
             - elsif !@edit[:new][:email][:to].present?
-              - emails = @edit[:new][:email][:to].sort { |a, b| a[1] <=> b[1] }
+              - emails = @edit[:new][:email][:to].sort
```

Comment:
I can't figure out how this code worked before.  Based on the callers, I believe `@edit[:new][:email][:to]` is just an Array of email addresses.  Given that, I think the old code basically sorted by the second character?!

```ruby
irb(main):001:0> ary = ["ac@x.com", "zb@x.com", "ba@x.com"]
=> ["ac@x.com", "zb@x.com", "ba@x.com"]
irb(main):002:0> ary.sort { |a, b| a[1] <=> b[1] }
=> ["ba@x.com", "zb@x.com", "ac@x.com"]
```

:confounded: